.eslintrc

```diff
@@ -20,10 +21,41 @@
     "import/prefer-default-export": "off",
     "camelcase": "off"
   },
-  "overrides": [{
-    "files": ["prelude/**/*"],
-    "rules": {
-      "strict": "off"
+  "overrides": [
```

Comment:
We turned a _lot_ of lint rules off here - why?

Reply:
These are the minimum set of specific rules that make linter work for the test/*.js only as of today.
Fixing them, all would generate a massive commit and is outside the scope of this PR.
I suggest that we remove the rules one by one later one, in a different PR to put all test files in compliance with the default lint rules.